src/version.js

```diff
@@ -1,6 +1,12 @@
-import React from 'react';
+/* eslint-disable no-var,object-shorthand */
 
-export const VERSION = React.version;
-export const REACT013 = VERSION.slice(0, 4) === '0.13';
-export const REACT014 = VERSION.slice(0, 4) === '0.14';
-export const REACT15 = VERSION.slice(0, 3) === '15.';
+var React = require('react');
+
+var VERSION = React.version;
+
+module.exports = {
```

Comment:
I'd prefer `npm run build` before the karma tests, so that the tests are always running on the build output.